EC-84: Restructure eZ Commerce doc, part 2 (#2)
ezsystems/commerce-documentation: docs/guide/basket/basket_features/basket_messages.md

# Basket messages

> Basket messages are stored in the Basket object, but they will not be stored in the database.

Reviewer: will not -> are not (repeated throughout the doc)

> Basket messages are stored in the Basket object, but they will not be stored in the database.

Reviewer 2: ```suggestion
Basket messages are stored in the Basket object, but they cannot be stored in the database.
```

> It is possible to store several success, error or notice messages for products.

Reviewer 2: ```suggestion
It is possible to store several success, errors or notice messages for products.
```

### Methods

|method|meaning|parameters|
|--- |--- |--- |

> |setSuccessMessage()|set one success message to the basket|$success|

Reviewer: Sets

> |getSuccessMessages()|get all success messages from the basket. If it is not set, an empty string will be returned.||

Reviewer: Capitalization of the initial verb. Also, "Gets", "Sets" or "Deletes" (repeated)

|setErrorMessage()|set one error message to the basket|$error|
|getErrorMessages()|get all error messages from the basket. If it is not set, an empty string will be returned.||
|setNoticeMessage()|set one notice message to the basket|$notice|
|getNoticeMessages()|get all notice messages from the basket. If it is not set, an empty string will be returned.||
|clearAllMessages()|delete all messages from the basket||
|removeSuccessMessageForSku()|delete all success messages for the given SKU from the success messages|$sku|

## Messages templating

> The messages parsed to the template through the `BasketController`, e.g. basket show:

Reviewer: What does "e.g. basket show" mean?
